code/modules/mob/living/simple_animal/hostile/beheaded_kamikaze.dm

```diff
@@ -0,0 +1,30 @@
+/mob/living/simple_animal/hostile/beheaded_kamikaze
+	name = "beheaded kamikaze"
+	desc = "You can't be serious."
+	icon_state = "clown" //placeholder
+	icon_living = "clown" //placeholder
```

Comment:
icon_dead should never exist if it always detonates on death. Icon_gib is just for specifying a unit gibbing animation